src/fuselage/CCPACSFuselageSections.h

```diff
@@ -43,6 +44,9 @@ class CCPACSFuselageSections : public generated::CPACSFuselageSections
     // Returns the section for a given index
     TIGL_EXPORT CCPACSFuselageSection& GetSection(int index) const;
 
+    //
+    TIGL_EXPORT std::map<std::string, CTiglFuselageSectionElement*> GetCTiglElements();
```

Comment:
Seems to be for me like a redundant helper function that should not be part of the interface of this class.

Reply:
removed and putted in fuselageHelper class